### PR TITLE
[Cognitive Services] SDK 7.4 for Cognitive Services

### PR DESCRIPTION
**Pull request by yangyuan**
Swagger PR: https://github.com/Azure/azure-rest-api-specs/pull/10722

### CODE REVIEW

**Comment by markcowl** [comment]
@yangyuan   Please follow the instructions [here](https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#sdk-generation-from-updated-spec) for re-generating your SDK from a new swagger.  It is not allowed to manually edit the generated code.

File: sdk/cognitiveservices/Microsoft.Azure.Management.CognitiveServices/src/Generated/Models/CognitiveServicesAccountApiProperties.cs [hunk 1]
```diff
@@ -40,12 +40,24 @@ public CognitiveServicesAccountApiProperties()
         /// flag to enable statistics of Bing Search.</param>
         /// <param name="storageAccountConnectionString">(Personalization Only)
         /// The storage account connection string.</param>
-        public CognitiveServicesAccountApiProperties(string qnaRuntimeEndpoint = default(string), bool? statisticsEnabled = default(bool?), string eventHubConnectionString = default(string), string storageAccountConnectionString = default(string))
+        /// <param name="aadClientId">(Metrics Advisor Only) The Azure AD
```
Comment (markcowl) [inline sdk/cognitiveservices/Microsoft.Azure.Management.CognitiveServices/src/Generated/Models/CognitiveServicesAccountApiProperties.cs:43]: Editing this file by hand is not allowed.  Please see : https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#sdk-generation-from-updated-spec  for how to regenerate your adk from an updated swagger.  You will also need to ensure the merged swagger PR is referenced in the comments for your PR

**Review by markcowl — DISMISSED** [review]

File: sdk/cognitiveservices/Microsoft.Azure.Management.CognitiveServices/src/Generated/Models/CognitiveServicesAccountApiProperties.cs (hunk 1 above)
Comment (yangyuan) [reply sdk/cognitiveservices/Microsoft.Azure.Management.CognitiveServices/src/Generated/Models/CognitiveServicesAccountApiProperties.cs:43]: Hi mark, this is auto generated. I didn't check-in all changes though.

**Comment by yangyuan** [comment]
@markcowl I've updated the PR, now it include full auto generated change, also updated tests and release notes.

**Review by bquantump — APPROVED** [review]

**Pull request merged**